scripts/run_inference.py

```diff
@@ -178,14 +181,31 @@ def load(
         try:
             with AdapterConfigPatcher(checkpoint_path, overrides):
                 try:
-                    model = AutoPeftModelForCausalLM.from_pretrained(
+                    if base_model_name_or_path is None:
+                        raise KeyError("base_model_name_or_path has to be passed")
```

Comment:
i think we shouldnt use `KeyError` here, it should be `ValueError`

Reply:
Done!